pkgs/servers/openafs/1.8/srcs.nix

```diff
@@ -1,16 +1,16 @@
 { fetchurl }:
 rec {
-  version = "1.8.12";
+  version = "1.8.12.2";
```

Comment:
The version still shows `1.8.12.2` should it not be `1.8.13`?

Reply:
Oops, fixed.